LibraryApp/src/components/form.js

```diff
@@ -57,8 +51,8 @@ class Form extends React.PureComponent {
       errors: {},
       isLoading: false,
     };
-
-    this.displayBooksList = this.displayBooksList.bind(this);
+    this.handleBackButtonClick = this.handleBackButtonClick.bind(this);
```

Comment:
again, let's try to move all back button related logic into a single file & then we add switch case to handle the rediction

Reply:
Done